lldb/source/Commands/CommandObjectDWIMPrint.cpp

```diff
@@ -173,7 +173,16 @@ bool CommandObjectDWIMPrint::DoExecute(StringRef command,
     auto *exe_scope = m_exe_ctx.GetBestExecutionContextScope();
     ValueObjectSP valobj_sp;
     ExpressionResults expr_result =
-        target.EvaluateExpression(expr, exe_scope, valobj_sp, eval_options);
+        target.EvaluateExpression(expr, exe_scope, valobj_sp, eval_options, &m_fixed_expression);
+    
+    // Only mention Fix-Its if the command applies them.
+    // The compiler errors can address any parsing issues after applying Fix-It(s).
+    if (!m_fixed_expression.empty() && target.GetEnableNotifyAboutFixIts()) {
+      Stream &error_stream = result.GetErrorStream();
+      error_stream.Printf("  Applying Fix-It to expression, changing it to:\n    %s\n",
+                          m_fixed_expression.c_str());
```

Comment:
You can use `error_stream << "..." << m_fixed_expression << "\n"` then you don't need to convert to c_str()

Reply:
NIce!
That actually makes the code cleaner too.

```cpp
if (!m_fixed_expression.empty() && target.GetEnableNotifyAboutFixIts()) {
  error_stream << "  Applying Fix-It to expression, changing it to:\n";
  error_stream << "    " << m_fixed_expression << "\n";
}
```